plugins/modules/gitlab_instance_variable.py

```diff
@@ -160,6 +166,7 @@ def create_variable(self, var_obj):
             "value": var_obj.get('value'),
             "masked": var_obj.get('masked'),
             "protected": var_obj.get('protected'),
+            "raw": var_obj.get('raw'),
```

Comment:
What does happen if the version of python-gitlab and/or GitLab itself do not support `raw`?

Reply:
As far as I can see python-gitlab just passes everything through from the API, so version of that would be irrelevant. If the gitlab instance is so old it doesn't support raw it's severely out of date (a couple of years) and long out of support.

That said I think what would happen is that you would get the same idempotence issue that currently exists for up to date versions.

Not sure if there is a very easy way to make this backwards compatible. And the gitlab_group_variable and gitlab_project_variable plugins do the exact same thing without trying to do any backwards compatibility handling.

Reply:
The support for these modules was added in e7d8ef4cf9183f59d79d758d3e3944c2b2324b0b. Since the PR is doing basically the same as here, let's assume this is fine as well.